app/javascript/mastodon/features/compose/components/compose_form.js

```diff
@@ -132,6 +132,10 @@ class ComposeForm extends ImmutablePureComponent {
     }
   }
 
+  componentDidMount () {
+    this.componentDidUpdate({ });
```

Comment:
Can you please extract `componentDidUpdate` into another method instead and then call it from both?